libs/core/kubeagi_core/embeddings/base.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 KubeAGI.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC, abstractmethod
+from typing import List
+
+
+class Embedding(ABC):
```

Comment:
@wangxinbiao Are there any differences with langchain Embedding???

Reply:
No difference, the OpenAI embeddings are directly used from Langchain's OpenAIEmbeddings.

Reply:
if so,we sholud remove the base definition